amp/WEB-INF/src/org/digijava/module/aim/util/ActivityVersionUtil.java

```diff
@@ -70,13 +71,15 @@ public static boolean implementsVersionable(Class[] array) {
         return ret;
     }
 
-    public static String generateFormattedOutput(HttpServletRequest request, Output out) throws WorkerException {
-        return generateFormattedOutput(request, out, null);
+    public static String generateFormattedOutput(Output out) throws WorkerException {
+        return generateFormattedOutput(out, null);
     }
 
-    public static String generateFormattedOutput(HttpServletRequest request, Output out, Output out1) throws WorkerException {
-        Site site = RequestUtils.getSite(request);
-        String langCode = RequestUtils.getNavigationLanguage(request).getCode();
+    public static String generateFormattedOutput(Output out, Output out1) throws WorkerException {
+        // Site site = RequestUtils.getSite(request);
```

Comment:
we can avoid commenting code so its easier to spot changes. Also if we need to reference the changed code we can use git history for this